eng/template/autorest.md.tpl

```diff
@@ -0,0 +1,10 @@
+### AutoRest Configuration
+
+> see https://aka.ms/autorest
+
+``` yaml
+require:
+- https://github.com/Azure/azure-rest-api-specs/blob/{{commitID}}/specification/{{rpName}}/resource-manager/readme.md
```

Comment:
Will these links actually exist initially?

Reply:
Yes, these must exist, otherwise the `autorest` command will complain it cannot get these files.